modules/hitl/src/backend/db.test.ts

```diff
@@ -0,0 +1,36 @@
+import Db, { TABLE_NAME_SESSIONS, TABLE_NAME_MESSAGES } from './db'
+import 'reflect-metadata'
+import Database from '../../../../packages/bp/src/core/database'
+
+import { createDatabaseSuite } from '../../../../packages/bp/src/core/database/index.tests'
+import { PersistedConsoleLogger } from '../../../../packages/bp/src/core/logger'
+import { createSpyObject, MockObject } from '../../../../packages/bp/src/core/misc/utils'
+
+const logger: MockObject<PersistedConsoleLogger> = createSpyObject<PersistedConsoleLogger>()
+
+createDatabaseSuite('Misunderstood - DB', (database: Database) => {
+  let db
+  beforeAll(async () => {
+    db = new Db({ database: database.knex, logger })
+    db.knex = database.knex
```

Comment:
as we discussed, is this monkey patching still useful ?

Reply:
Nope, I'll delete